src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const clickableImagesList = document.getElementById('thumbs');
+
+const largeImage = document.getElementById('largeImg');
+
+clickableImagesList.addEventListener('click', (eventOnClick) => {
+  // Prevent browser from navigating to the href
+  eventOnClick.preventDefault();
+
+  // Had some troubles with this.
+  /*
+    At first i've done this:
+    src = eventOnClick.target.getAttribite('src');
```

Comment:
There is a typo in the commented-out code: 'getAttribite' should be 'getAttribute'. This is not affecting the current implementation since it's commented out, but it should be corrected if you plan to use this code.